security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_generate_conf.php

```diff
@@ -1022,7 +1022,7 @@
 else {
 	$arpspoof_preproc .= "preprocessor arpspoof\n";
 }
-if (is_array($snortcfg['arp_spoof_engine']['item']) && count($snortcfg['arp_spoof_engine']['item']) > 0) {
+if (array_get_path($snortcfg, 'arp_spoof_engine/item')) {
 	foreach ($snortcfg['arp_spoof_engine']['item'] as $f => $v) {
```

Comment:
```suggestion
foreach(array_get_path($snortcfg, 'arp_spoof_engine/item', []) as $f => $v) {
```
Just is with array iteration using a default in config_get_path(), the same can be done with array_get_path() if you want to eliminate an extra scope level here

Reply:
Yep, using the default return value simplifies the code. Made the suggested changes.